graphql/util.go

```diff
@@ -1290,3 +1290,37 @@ func getAPISubscriptionsForProject(ctx context.Context, projectId string) ([]*re
 	}
 	return res, nil
 }
+
+func GetProjectSettingsSection(ctx context.Context, sectionName ProjectSettingsSection) model.ProjectPageSection {
+	var section model.ProjectPageSection
+	switch sectionName {
+	case ProjectSettingsSectionGeneral:
```

Comment:
Would it work to switch model.ProjectPageSection to just be the same enum?

Reply:
Just looked into it briefly, this pattern has not been used elsewhere and also introduces a dependency cycle (which tbh is perhaps the reason why). I could spend some more time looking into this but it may be more trouble than it's worth!